feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.ts

```diff
@@ -21,7 +25,7 @@ import { ConfiguratorAttributeNumericInputFieldService } from './configurator-at
 })
 export class ConfiguratorAttributeNumericInputFieldComponent
   extends ConfiguratorAttributeBaseComponent
```

Comment:
Can this component extends `ConfiguratorAttributeInputFieldComponent`?

Reply:
picked

Reply:
Picked! I will check it

Reply:
@WeizhengSap, I created a [PR](https://github.com/SAP/spartacus/pull/11946) regarding this topic, could you have a look at it? Many thanks in advance!

Reply:
Done!